common/src/test/java/com/revenuecat/purchases/common/DeviceCacheTest.kt

```diff
@@ -126,6 +126,33 @@ class DeviceCacheTest {
         assertThat(info?.entitlements?.verification).isEqualTo(VerificationResult.NOT_REQUESTED)
     }
 
+    @Test
+    fun `given a valid purchaser info without request date, the JSON is parsed correctly`() {
+        val deprecatedValidCachedCustomerInfo by lazy {
+            JSONObject(Responses.validFullPurchaserResponse).apply {
+                put("schema_version", PURCHASER_INFO_SCHEMA_VERSION)
```

Comment:
It would be nice to finish renaming all these to "customer info". Purchaser info was the old name